File: LinkedList/linkedList.py
class Node:
    def __init__(self,item):
        self.data=item
        self.next=None

class LinkedList:
    def __init__(self):
        self.head=None

if __name__=='__main__':
    ll=LinkedList()

    ll.head=Node(100)
    two=Node(200)
    three=Node(300)

    ll.head.next=two
    two.next=three 

    while ll.head!=None:
        print(ll.head.data)
        ll.head=ll.head.next               

File: LinkedList/linkedList.js
class Node{
    constructor(item){
        this.data=item
        this.next=null
    }
}

class LinkedList{
    constructor(){
        this.head=null
    }
}

let ll=new LinkedList()
ll.head=new Node(15)
let two=new Node(25)
let three=new Node(35)

ll.head.next=two
two.next=three

while(ll.head!=null){
    console.log(ll.head.data)
    ll.head=ll.head.next
}